starwars/src/components/card.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./StarWars.css";
+
+const Card = props => {
+  return (
+    <div className="card-container">
+      {props.data.map(character => {
+        return (
+          <div className="card">
```

Comment:
Don't neglect passing a `key` attribute to list of jsx elements returned by `.map`